settings/memcache.settings.php

```diff
@@ -85,5 +85,8 @@
 
     // Move locks to memcache.
     $settings['container_yamls'][] = DRUPAL_ROOT . '/../vendor/acquia/blt/settings/memcache.yml';
+
+    // Enable compression for PHP 7.
+    $settings['memcache']['options'][Memcached::OPT_COMPRESSION] = TRUE;
```

Comment:
FYI compression was enabled by default in https://www.drupal.org/project/memcache/releases/8.x-2.0-alpha7